extensions/2.0/Khronos/KHR_interactivity/Specification.adoc

```diff
@@ -0,0 +1,4196 @@
+.... validate the `value` property value according to the <<inline-value-validation,Inline Value Validation>> using the `type` property.
+
+[[validation-declaration-object]]
+== Declaration Object Validation
+
+1. _Assert_ that the element of the `declarations` array is a JSON object ("`the declaration`").
+2. _Assert_ that "`the declaration`" object has the `op` property that is a JSON string.
+3. If "`the declaration`" object does not have the `extension` property:
+.. if the `op` property value does not match any operation defined in this Specification,
+... _reject the graph_;
+.. if "`the declaration`" object has the `inputValueSockets` and/or `outputValueSockets` properties,
+... _reject the graph_;
+4. If "`the declaration`" object has the `extension` property:
+.. _assert_ that the `extension` property is a JSON string;
+.. if the "`the declaration`" object has the `inputValueSockets` property:
+... _assert_ that the `inputValueSockets` property is a non-empty JSON object;
+... if the `inputValueSockets` object has more properties than the implementation-specific limit on the number of input value sockets for declarations,
+.... _reject the graph_;
+... _assert_ that the graph has the `types` property;
+... for each property of the `inputValueSockets` JSON object:
+.... _assert_ that the property is a JSON object ("`the input value socket declaration`");
+.... _assert_ that "`the input value socket declaration`" object has the `type` property that is a _JSON index_;
+.... if the `type` property value is not less than the `types` graph array length,
+..... _reject the graph_;
+.. if "`the declaration`" object has the `outputValueSockets` property:
+... _assert_ that the `outputValueSockets` property is a non-empty JSON object;
+... if the `outputValueSockets` object has more properties than the implementation-specific limit on the number of output value sockets for declarations,
+.... _reject the graph_;
+... _assert_ that the graph has the `types` property;
+... for each property of the `outputValueSockets` JSON object:
+.... _assert_ that the property is a JSON object ("`the output value socket declaration`");
+.... _assert_ that "`the output value socket declaration`" object has the `type` property that is a _JSON index_;
+.... if the `type` property value is not less than the `types` graph array length,
+..... _reject the graph_.
+
+[[validation-node-object]]
+== Node Object Validation
+
+1. _Assert_ that the element of the `nodes` array is a JSON object ("`the node`").
+2. _Assert_ that "`the node`" object has the `declaration` property that is a JSON index.
+3. If the `declaration` property value is not less than the `declarations` graph array length,
+.. _reject the graph_.
+4. If "`the node`" object has the `configuration` property:
+.. _assert_ that the `configuration` property is a non-empty JSON object;
+.. _assert_ that each property of the `configuration` JSON object is a JSON array.
```

Comment:
Each property of the configuration object should have a `value` property that is an array

Reply:
Good catch! Fixed.